src/subdomains/generic/kyc/services/integration/kyc-document.service.ts

```diff
@@ -2,6 +2,7 @@ import { Injectable, UnsupportedMediaTypeException } from '@nestjs/common';
 import { AzureStorageService, BlobContent } from 'src/integration/infrastructure/azure-storage.service';
 import { UserData } from 'src/subdomains/generic/user/models/user-data/user-data.entity';
 import { FileType, KycFile } from '../../dto/kyc-file.dto';
```

Comment:
Maybe I would rename the `KycFile` in the kycFileDto file to `KycFileInterface` or `KycFileDto` 🤔